runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -686,7 +686,18 @@
 	/**
 	 * 1 is true, 0 is false.
 	 */
-	GAUNTLET_ENTERED(9178);
+	GAUNTLET_ENTERED(9178),
```

Comment:
Can you remove the sculliscep varbits from here too?

Reply:
`GAUNTLET_ENTERED(9178),
 @pklite
pklite 4 minutes ago Member
Can you remove the sculliscep varbits from here too?

Reply…

	/**
	 * 1 if full of Mushrooms, 0 is empty.
	 */
	MUSHY_PIT(5809),

	/**
	 * Current location of Sulliuscep tree.
	 */
	SULLIUSCEP_LOCATION(5808);

`

lmao i hate githubs web edit interface but if you remove these i think it will resolve the conflicts